docs/source/control_tutorials.rst

```diff
@@ -0,0 +1,349 @@
+.. py:module::rig.machine_control
+
+Machine Control Tutorials
+=========================
+
+SpiNNaker machines consist of a network of SpiNNaker chips and, in larger
+systems, a set of Board Management Processors (BMPs) which control and monitor
+systems' power and temperature. SpiNNaker (and BMPs) are controlled using `SCP`_
+packets (a protocol built on top of `SDP`_) sent over the network to a machine.
+Rig includes a set of high-level wrappers around the low-level SCP commands
+which are tailored towards SpiNNaker application developers.
+
+.. _SCP: https://spinnaker.cs.man.ac.uk/tiki-download_wiki_attachment.php?attId=17&page=Application%20note%205%20-%20SCP%20Specification&download=y
+
+.. _SDP: https://spinnaker.cs.man.ac.uk/tiki-download_wiki_attachment.php?attId=16&page=Application%20note%204%20-%20SDP%20Specification&download=y
+
+.. note::
+    Rig does not aim to provide a complete Python implementation of the full
+    (low-level) SCP command set. Users who encounter missing functionality as a
+    result of this are encouraged to submit a patch or open an issue as the
+    developers are open to (reasonable) suggestions!
+
+The two high-level interfaces are:
+
+:py:class:`.MachineController`
+    Interact with and control SpiNNaker chips, e.g. boot, load applications,
+    read/write memory.
+:py:class:`.BMPController`
+    Interact with and control BMPs, e.g. control power-supplies, monitor
+    system temperature, read/write FPGA registers. Only applicable to machines
+    based on SpiNN-5 boards.
+
+A tutorial for each of these interfaces is presented below.
+
+For those wishing to extend these interfaces or directly use SCP or SDP packets,
+the underlying advanced APIs are also very briefly introduced.
+
+:py:class:`.MachineController` Tutorial
+---------------------------------------
+
+To get started, lets instantiate a :py:class:`.MachineController` which is as
+simple as giving the hostname or IP address of the machine::
+
+    >>> from rig.machine_control import MachineController
+    >>> mc = MachineController("spinnaker_hostname")
+
+.. note::
+    If you're using a multi-board machine, give the hostname of the (0, 0) chip.
+    Support for connecting to multiple Ethernet ports of a SpiNNaker machine is
+    not currently available but should be automatic. 
+
+Booting
+^^^^^^^
+
+You can :py:meth:`~.MachineController.boot` boot the system like so::
```

Comment:
This will format as "You can __boot(...)__ boot the system like so"... which seems silly.